src/test/scala/is/hail/io/ImportMatrixSuite.scala

```diff
@@ -6,62 +6,118 @@ import is.hail.check.Gen
 import is.hail.check.Prop.forAll
 import is.hail.expr._
 import is.hail.expr.types._
+import is.hail.table.Table
 import is.hail.variant.{MatrixTable, VSMSubgen}
 import org.apache.spark.SparkException
 import org.testng.annotations.Test
 import is.hail.utils._
 
 class ImportMatrixSuite extends SparkSuite {
 
+  val genMatrix = VSMSubgen(
+    sSigGen = Gen.const(TString()),
+    saSigGen = Gen.const(TStruct.empty()),
+    vSigGen = Gen.const(TString()),
+    vaSigGen = Gen.const(TStruct.empty()),
+    globalSigGen = Gen.const(TStruct.empty()),
+    tSigGen = Gen.zip(Gen.oneOf[Type](TInt32(), TInt64(), TFloat32(), TFloat64(), TString()), Gen.coin(0.2))
+      .map { case (typ, req) => typ.setRequired(req) }.map { t => TStruct("x" -> t) },
+    sGen = (t: Type) => Gen.identifier.map(s => s: Annotation),
```

Comment:
This seems dangerous. What happens if the `sSig` isn't a `TString()`?

Reply:
It is, I think? sSigGen and vSigGen are Gen.const(TString()).

Reply:
(I made vSigGen more generic, and changed vGen--LoadMatrix necessarily imports sampleIDs as TString() right now.)